src/pages/events/[id]/index.tsx

```diff
@@ -173,6 +178,13 @@ const EventView: NextPage<{ json: string }> = ({ json }) => {
 											No description was provided for this event.
 										</p>
 									)}
+									{/*
+										This section can't be rendered on the server as it uses .toLocaleString.
+										If I remember correctly, this causes a hydration mismatch error at minimum, and even if bypassed, it
+										won't render a user-locale accurate datetime string.
+										It's possible a custom formatter could be used to get around this, but I'm not sure it's necessary.
+										If switching off NoSSR, use date-fns as the server date formatter.
+									*/}
```

Comment:
This comment was mostly made to prevent a regression should we switch off `NoSSR` in the future; I don't believe at this moment that a change in rendering is required or beneficial. The performance will be identical, whether it's rendered SSR or Client.